spec.md

```diff
@@ -864,6 +867,31 @@ message GetCapacityResponse {
 }
 ```
 
+#### `ProbeController`
+
+A Controller Plugin MUST implement this RPC call.
+The Plugin SHALL assume that this RPC will be executed on the controller which could be either a central service or a distributed one.
```

Comment:
I'd rather just replace this line with something a bit different:
The CO SHALL invoke this RPC prior to any other controller service RPC in order to allow the CO to determine the readiness of the controller service.
A CO MAY invoke this call multiple times with the understanding that a plugin's implementation MAY NOT be trivial and there MAY be overhead incurred by such repeated calls.